qulice-maven-plugin/src/site/apt/example-exclude.apt.vm

```diff
@@ -70,3 +71,13 @@ Exclude
   folders;
   2) <<<codenarc:**/src/ex1/Main.groovy,**/src/ex2/Main2.groovy>>> excludes
   <<<Main.groovy>>> and <<<Main2.groovy>>> in appropriate packages.
+
+  Exclude for Findbug is a bit different, as expression must be exact class name
```

Comment:
slightly reworded that: `..as expression has to be either an exact class name or start with the..`